functions.yml

```diff
@@ -132,11 +140,13 @@ complex_type_manipulation:
       args:
         - name: vector
           doc: Vector to convert
+      returns: An array of doubles
 
 etl:
   functions:
     - name: hard_calls
-      doc: Converts an array of probabilities to hard calls
+      doc: Converts an array of probabilities to hard calls. See :ref:`variant-data-transformations` 
```

Comment:
Converts an array of probabilities to hard calls. The probabilites are assumed to be diploid. See :ref:`variant-data-transformations`.

Reply:
Changed